src/SimpleCalculator.Test/CalculationService.Tests/CalculationServiceTests.cs

```diff
@@ -43,9 +43,10 @@ public void AddToDbService_Should_Save_Entities()
 
         mockContext.Object.CalculationResultEntities.Add(entities);
         var service = new CalculationService(_multiplicationServiceMock.Object, _divisionServiceMock.Object, _subtractionServiceMock.Object, _additionServiceMock.Object, mockContext.Object, _mapper);
-        service.AddToDbService(model, passedEnum);
+        service.AddToDb(model, passedEnum);
+        var data = new CalculationResultEntity {Id = 1, MathOperator = passedEnum.ToString() ,FirstNumber = 12, SecondNumber = 10, Result = 22};
 
-        mockDbSet.Verify(x => x.Add(It.IsAny<CalculationResultEntity>()), Times.Once());
+        mockDbSet.Verify(x => x.Add(data), Times.Never());
```

Comment:
Does `Times.Never()` mean that it shouldn't be executed at all? Isn't it opposite what you are testing there?